packages/rollup-config/index.js

```diff
@@ -0,0 +1,77 @@
+const { resolve } = require('path')
+const { terser } = require('rollup-plugin-terser')
+
+const replace = require('rollup-plugin-replace')
+const typescript = require('rollup-plugin-typescript')
+const commonjs = require('rollup-plugin-commonjs')
+
+const camelize = require('./lib/camelize')
+
+const env = process.env.NODE_ENV
+const pkg = require(resolve('package.json'))
+const pkgNamespace = camelize(pkg.name.split('/')[1])
+const pkgDependencies = [
+	...Object.keys(pkg.dependencies || {}),
+	...Object.keys(pkg.peerDependencies || {}),
+]
+
+const input = 'src/index.ts'
+
+const plugins = [
+	replace({
+		'process.env.NODE_ENV': JSON.stringify(env),
+	}),
+	commonjs({
+		include: /node_modules/,
+	}),
+	typescript(),
+]
+
+const productionPlugins = [...plugins, terser()]
+
+module.exports = ({
+	namespace,
+	cjs = true,
+	esm = true,
+	iife = !!pkg.browser,
+	sourcemap = true,
+} = {}) => {
+	const name = 'rematch.' + namespace || pkgNamespace
+
+	const rollupBuild = {
+		input,
+		plugins: env === 'production' ? productionPlugins : plugins,
+		external: pkgDependencies,
+		output: [],
+	}
+
+	if (iife) {
```

Comment:
This is a big change. UMD is just more weight if this is only our "Browser" file, since UMD includes cjs.